source/includes/_accounts.md

```diff
@@ -120,3 +132,74 @@ A successful GET will return an HTTP 200
 <aside class="warning">
 An unsuccessful GET will return an HTTP 404
 </aside>
+
+## Update a specific account
+
+This endpoint updates an individual account
```

Comment:
Should we specify that giving specific interval or summary usages overwrites?

Reply:
Yeah that's a good idea